content/page/supporters.en.md

```diff
@@ -4,50 +4,34 @@ date: 2018-08-17 17:03:06
 draft: false
 ---
 
-# Our partners
-<div class="row justify-content-md-center">
-    <div class="col-md-4 text-center">
-        <img src="/media/partners/fdroid.svg" width="120px" height="120px" class="mt-3 ml-auto mr-auto"/>
-        <div class="card-body">
-            <h6 class="card-title"><a href="https://f-droid.org/">F-Droid</a></h6>
-        </div>
-    </div>
-    <div class="col-md-4 text-center">
-        <img src="/media/partners/yale-privacy-lab.png" width="120px" height="120px"class="mt-3 ml-auto mr-auto"/>
-        <div class="card-body">
-            <h6 class="card-title"><a href="https://privacylab.yale.edu/">Yale Privacy Lab</a></h6>
-        </div>
-    </div>
-</div>
-
```

Comment:
I think we need to keep YPL and Fdroid as our supporters and not remove them completely.